### ISSUE TITLE
"Too many arguments for format string" warning on every failure with $VERBOSE = true

### ISSUE THREAD

**Issue by unikitty37** [djezzzl/database_consistency#156]
I recently updated from Ruby 2.7 to Ruby 3.1; as a part of this, I set `$VERBOSE = true` in the development environment.

I now get a warning with every error thrown by database_consistency:

```
/usr/local/bundle/gems/database_consistency-1.3.9/lib/database_consistency/writers/simple/base.rb:50: warning: too many arguments for format string
ColumnPresenceChecker fail User password column (password) is missing in table (users) but used for presence validation
/usr/local/bundle/gems/database_consistency-1.3.9/lib/database_consistency/writers/simple/base.rb:50: warning: too many arguments for format string
ColumnPresenceChecker fail Event calendar_feed association foreign key column should be required in the database
```

Unfortunately, that's in a fairly generic method, and warnings don't generate full stack traces, so I can't work out where the format string is coming from 😞

**Comment by djezzzl** [comment]
Hi @unikitty37,

Thank you for letting me know! I will improve it as soon as possible.

**Comment by djezzzl** [comment]
Hey, I've just released 1.4.1. Could you please try and let me know if that helped? 

Feel free to reopen an issue if needed.

**Comment by unikitty37** [comment]
Thanks — that's fixed it! 👍
